.github/workflows/release_linux_aarch64.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     env:
       # setting up python and docker image
       py: /opt/python/${{ matrix.python-version }}/bin/python
-      img: quay.io/pypa/manylinux2014_aarch64
+      img: docker://quay.io/pypa/manylinux2014_aarch64@a2d8ba969c23fb91e383b99f62324769944fe2518404b56b186037bab7ad1117
```

Comment:
```suggestion
      img: docker://quay.io/pypa/manylinux2014_aarch64:latest@sha256:a2d8ba969c23fb91e383b99f62324769944fe2518404b56b186037bab7ad1117
```

Reply:
maybe we ignore these Docker files in environments for now. Somehow the syntax doesn't seem to fit quite right. 
For the score at scorecard they don't seem to be detected, and then dependabot should also find them as stale. If one has the correct syntax, I would perhaps test this with the second last hash if we then get an automatic update suggestion.